app/src/main/java/org/listenbrainz/android/data/sources/brainzplayer/Playable.kt

```diff
@@ -0,0 +1,8 @@
+package org.listenbrainz.android.data.sources.brainzplayer
+
+data class Playable(
+    val type: String,
+    val id: Long,
+    val songs: List<Song>,
+    val currentlyPlayingSongIndex: Int
```

Comment:
Change this to `currentSongIndex` I suppose

Reply:
Done!